content/ch-algorithms/deutsch-jozsa.ipynb

```diff
@@ -193,50 +193,52 @@
    "source": [
     "## 2. Worked Example  <a id='example'></a>\n",
     "\n",
-    "Let's go through a specific example for a two bit balanced function, where we apply X-gates before both CNOTs.\n",
+    "Let's go through a specific example for a two bit balanced function: \n",
     "\n",
     "<ol>\n",
     "    <li> The first register of two qubits is initialized to $|00\\rangle$ and the second register qubit to $|1\\rangle$ \n",
+    "        \n",
+    "(Note that we are using subscripts 1, 2, and 3 to index the qubits. A subscript of \"12\" indicates the state of the register containing qubits 1 and 2)\n",
```

Comment:
Should this maybe be 0, 1, 2 instead of 1, 2, 3?

Reply:
Hmm I don't see a problem really, as long as it's consistent throughout the example. In fact 0-indexing is sometimes unintuitive for readers from other backgrounds. But that's your call to take :) I can change it if you feel that would be better.

Reply:
Let's leave it 🙂

Reply:
Great 🙂